jupyter_conda/_version.py

```diff
@@ -1,2 +1,2 @@
-version_info = (3, 4, 1)
+version_info = (4, 0, 0)
```

Comment:
It's probably a matter of style and preference, but should the version bump be done in this PR or in a separate release commit?

Reply:
This is a good idea.